### ISSUE TITLE
Lighthouse baselines - old and new DL sites

### ISSUE THREAD

**Issue by mark-dce** [yalelibrary/YUL-DC#314]
**STORY**
As a repository owner, I'd like to run google lighthouse against my site, so that I have a basic idea of how accessible my site is.

**NOTES**
Lighthouse is not a full accessibility test suite, but it's a quick and lightweight tool that anyone one the team can use to discover broad accessibility issues that should be addressed prior to conducting more detailed accessibility audits.

Main lighthouse site https://developers.google.com/web/tools/lighthouse

Here's a document that describes the way DCE used lighthouse for UCLA - In particular see Appendix A - Step-by-step Test Procedure for how we ran lighthouse https://drive.google.com/file/d/1l2_P06XHZ1fKCTsl2aC81jCIqOAWAeAn/view?usp=sharing  

This folder contains the original screen captures as well
https://drive.google.com/drive/folders/1LcxB_M5n7O_4bD-zP1fVVXrv6xQFJzKV

## Acceptance Criteria
For the old ( https://brbl-dl.library.yale.edu/vufind/ ) and new ( https://collections-demo.curationexperts.com/ ) sites
- [x] Run Lighthouse on the Home Page
- [x] Run Lighthouse on the search results page 
- [x] Run Lighthouse on the Object Show Page

For the new site
- [x] Write up remediation tickets for any page that have an Accessibility score below 80; or any easy-to-fix problems

**Comment by K8Sewell** [comment]
# Sumarry
## Collections Demo
### Home Page 
Performance - 19
Accessibility - 90
Best Practices - 100
SEO - 89
### Search Results Page
Performance - 16
Accessibility - 90
Best Practices - 100
SEO - 89
### Object Show Page
Performance - 4
Accessibility - 80
Best Practices - 93
SEO - 80

## Existing Site
### Home Page 
Performance - 40
Accessibility - 91
Best Practices - 79
SEO - 90
### Search Results Page
Performance - 26
Accessibility - 81
Best Practices - 79
SEO - 90
### Object Show Page
Performance - 12
Accessibility - 80
Best Practices - 71
SEO - 70

Link to Research w/ Screenshots - https://docs.google.com/document/d/1RH3t0ZibKZYY88rG1btm4ydes_Ce7hWjGdRYKwDZP_M/edit?usp=sharing
